src/iotjs_binding.c

```diff
@@ -284,7 +278,7 @@ jerry_value_t iotjs_jhelper_call(jerry_value_t jfunc, jerry_value_t jthis,
   jerry_length_t jargc_ = iotjs_jargs_length(jargs);
 
 #ifdef NDEBUG
-  jargv_ = (jerry_value_t*)jargs->unsafe.argv;
+  jargv_ = (jerry_value_t*)jargs->argv;
```

Comment:
@knightburton What about removing the code enclosed with `#ifdef NDEBUG`? We don't need this line because member `unsafe` is eliminated.

Reply:
+1